fiftyone/core/dataset.py

```
@@ -10296,6 +10612,37 @@ def _save_view(view, fields=None):
+                            "_media_reference": {"$exists": True},
+                        },
+                        {
+                            "filepath": {"$exists": False},
+                            "_media_reference": {"$exists": False},
+                        },
+                        {"_media_reference.kind": {"$ne": reference_kind}},
+                    ]
+                }
+            },
+            {"$limit": 1},
+        ]
```

Ruff (0.16.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)